tensorflow/python/kernel_tests/array_ops/depthtospace_op_test.py

```diff
@@ -77,6 +77,13 @@ def testBasicFloat16(self):
     x_out = [[[[1], [2]], [[3], [4]]]]
     self._testOne(x_np, block_size, x_out, dtype=dtypes.float16)
 
+  @test_util.run_deprecated_v1
```

Comment:
Rather than duplicate the logic, can you either convert to a loop over dtypes like some of the other tests, or better yet a parameterized test (like [here](https://github.com/tensorflow/tensorflow/blob/da55f2228f3279495dd689bd5514d8ec7a78da3b/tensorflow/python/kernel_tests/array_ops/shape_ops_test.py#L562) for an example).

Reply:
Thank you @cantonios for the review! I've parameterized the tests.